src/index.js

```diff
@@ -0,0 +1,74 @@
+import './style.css';
+import List, { editTask, cleanAll } from './crud.js';
+import state from './interactive.js';
+
+const add = document.getElementById('send');
```

Comment:
OPTIONAL
It's important to note that “add” could be renamed to something more descriptive, like “addButton” to enhance code readability.